components/AnnouncementCard.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
-
-import '../scss/announcementCard.scss'
+import styles from '../scss/announcementCard.module.scss'
 
 type Props = {
   announcements: string[]
 }
 
 const AnnouncementCard: React.FC<Props> = ({ announcements }) => {
   return (
-    <div className="announcement-card__container card shadow-sm mt-3 p-2 d-flex border-0">
+    <div
+      className={`${styles['announcement-card__container']} card shadow-sm mt-3 p-2 d-flex border-0`}
```

Comment:
I choose to use camelCase everywhere because otherwise you need to change most css names (you can't use `styles.announcement-card__container` and etc). It's rather verbose but on the bright side it's clear that this rule comes from custom css file and not from bootstrap.